pkg/ccl/streamingccl/replicationtestutils/testutils.go

```diff
@@ -277,6 +298,16 @@ func (c *TenantStreamingClusters) BuildCreateTenantQuery(externalConnection stri
 	return streamReplStmt
 }
 
+// DefaultAppTenantTestingKnobs returns the default testing knobs for an application tenant.
+func DefaultAppTenantTestingKnobs() base.TestingKnobs {
+	jobTestingKnobs := jobs.NewTestingKnobsWithShortIntervals()
+	jobTestingKnobs.SchedulerDaemonInitialScanDelay = func() time.Duration { return time.Second }
+	jobTestingKnobs.SchedulerDaemonScanDelay = func() time.Duration { return time.Second }
```

Comment:
I suppose you could put this in a defaultJobTestingKnobs() that could be reused below as well.

Reply:
Good point. Done.